README.md

```diff
@@ -1,56 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# React-rails-hello
 
-> Description the project.
+> A simple repo for testing how to run React in a Ror project. It consists on a react component rendering a random greeting message fetched from the '/greetings' API endpoint created with rails. 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby on rails
+- React
+- Webpack
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+Rails 7
+Ruby v 3.1.2
+
 ### Setup
 
-### Install
+- Clone this repo and open it locally
 
-### Usage
+### Install
 
-### Run tests
+- npm i
+- bundle i
 
```

Comment:
- [ ] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes
- `git clone URL` , `cd project_name`,
- `bundle install`
- `rails db:create db:migrate db:seed`
- `yarn install`
- `yarn build`
- `./bin/dev`
etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.